docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     ports:
       - "3306:3306"
   postgres:
-    image: postgres:9.6
+    image: postgres:10.21
```

Comment:
Previous test failure was related to some postgres bug. As `9.6` is not supported anymore, I bumped to a supported version.